Assingment_1-2/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+using namespace std;
+
+int main() {
+
+    const int SHARES_BOUGHT = 1000;
+    const double PURCHASE_PRICE = 45.50;
+    double money_spent = double(SHARES_BOUGHT*PURCHASE_PRICE);
+    std::cout << "Amount of money spent on buying stocks = " << double(money_spent) << std::endl;
+    const double COMMISSION_RATE = 0.02;
+    double commission_fee_buy = double(SHARES_BOUGHT*PURCHASE_PRICE*COMMISSION_RATE);
+    std::cout << "Amount of money payed to the broker when buying stocks = " << double(commission_fee_buy) << std::endl;
+    const int SHARES_SOLD = 1000;
+    const double SALE_PRICE = 56.90;
```

Comment:
While the vertical distance between declaration and usage is important constants should go to the top of the file because they are immutable and its ok for them to be used all over your program because they will not change